packages/components/src/components/radio-button/radio-button.stories.ts

```diff
@@ -31,142 +38,98 @@ export default {
  */
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- * Use the `size` attribute to change the size of the input radio. This attribute affects the font-size within the element, while the element itself remains the same size.
+ * Use the `size` attribute to change the size.
  */
 
 export const Size = {
-  parameters: { controls: { exclude: ['size'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+  name: 'Size',
+  render: () => html`
+    <div class="flex gap-12">
```

Comment:
Please center the button:

Figma:
<img width="948" alt="image" src="https://github.com/user-attachments/assets/66541dc0-0b84-4c1a-b688-cd10f31baa24">